portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormPortletConfiguration.path

```diff
@@ -14,6 +14,12 @@
 	<td>xpath=(//div[contains(@id,'SearchContainer')]//tr[contains(.,'${key_formName}')]/td//a)[1]</td>
 	<td></td>
 </tr>
+<!--TRANSLATE-->
+<tr>
+	<td>SELECT_LANGUAGE</td>
+	<td>//span[contains(text(),'${changeLanguageTo}')]</td>
```

Comment:
@wesleyroberts, please rename:

- `changeLanguageTo` to `key_language`

Reply:
changes made